project/MultiScalaProject.scala

```diff
@@ -0,0 +1,96 @@
+package build
+
+import sbt._
+import Keys._
+
+final class MultiScalaProject private (private val projects: Map[String, Project]) {
+  import MultiScalaProject._
+
+  def settings(ss: Def.SettingsDefinition*): MultiScalaProject =
+    copy(_.settings(ss: _*))
+
+  def enablePlugins(ns: Plugins*): MultiScalaProject =
+    copy(_.enablePlugins(ns: _*))
+
+  def dependsOn(deps: ScopedMultiScalaProject*): MultiScalaProject = {
+    def cpd(d: ScopedMultiScalaProject): Map[String, ClasspathDependency] =
+      strictMapValues(d.project.projects)(ClasspathDependency(_, d.configuration))
+
+    val depsByVersion: Map[String, Seq[ClasspathDependency]] =
+      strictMapValues(deps.flatMap(cpd).groupBy(_._1))(_.map(_._2))
+    zipped(depsByVersion)(_.dependsOn(_ :_*))
+  }
+
+  def configs(cs: Configuration*): MultiScalaProject =
+    copy(_.configs(cs: _*))
+
+  def zippedSettings(that: MultiScalaProject)(ss: Project => Def.SettingsDefinition): MultiScalaProject =
+    zipped(that.projects)((p, sp) => p.settings(ss(sp)))
+
+  def zippedSettings(project: String)(ss: LocalProject => Def.SettingsDefinition): MultiScalaProject =
+    zippedSettings(Seq(project))(ps => ss(ps(0)))
+
+  def zippedSettings(pn0: String, pn1: String)(
+      ss: (LocalProject, LocalProject) => Def.SettingsDefinition): MultiScalaProject =
+    zippedSettings(Seq(pn0, pn1))(ps => ss(ps(0), ps(1)))
+
+  def zippedSettings(projectNames: Seq[String])(
+      ss: Seq[LocalProject] => Def.SettingsDefinition): MultiScalaProject = {
+    val ps = for {
+      (v, p) <- projects
+    } yield {
+      val lps = projectNames.map(pn => LocalProject(projectID(pn, v)))
+      v -> p.settings(ss(lps))
+    }
+    new MultiScalaProject(ps)
+  }
+
+  def %(configuration: String) = new ScopedMultiScalaProject(this, Some(configuration))
+
+  def v(scalaVersion: String): Project = projects(scalaVersion)
+
+  def allProjects(): Iterable[Project] = projects.values
```

Comment:
```suggestion
  def allProjects: Iterable[Project] = projects.values
```
(especially since you seem to be *using* it without the `()`)